<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SketchDuplicate</class>
 <widget class="QWidget" name="SketchDuplicate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { background: rgba(22, 25, 27,220); }

QPushButton { 
	background:rgb(90,90,90,128);
	border: 3px solid rgb(90,90,90,0);
	padding: 5px;
    color: white;
	font-size: 12px;
	text-align:left;
}

QCheckBox { 
	background:rgb(27, 30, 32,128);
	border: 3px solid rgb(90,90,90,0);
	padding: 5px;
    color: white;
	font-size: 12px;
}

QLabel, QRadioButton{
	color: white;
	background: transparent;
	min-width: 60px;
}

QSpinBox, QDoubleSpinBox{
	color:white;
}

QPushButton:hover{ background:rgb(255,153,0); border-color:rgb(255,153,0); }
QPushButton:pressed{ background:rgb(255,153,0); border-color:#FFB444; top:2px; }

QGroupBox{ color:white; }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="dupT">
     <property name="styleSheet">
      <string notr="true">QGroupBox{background:rgb(255,0,0,20);}</string>
     </property>
     <property name="title">
      <string>Translational</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QSpinBox" name="tCount">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Count</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Delta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="tY">
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="tX">
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="tZ">
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dupRef">
     <property name="styleSheet">
      <string notr="true">QGroupBox{background:rgb(0,255,0,20);}</string>
     </property>
     <property name="title">
      <string>Reflectional</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="2">
       <widget class="QRadioButton" name="refY">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="refX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="refOffset">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-100.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QRadioButton" name="refZ">
        <property name="text">
         <string>Z</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0" rowspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dupRot">
     <property name="styleSheet">
      <string notr="true">QGroupBox{background:rgb(0,0,255,20);}</string>
     </property>
     <property name="title">
      <string>Rotational</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Count</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Axis</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rotX">
        <property name="text">
         <string>X</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="rot_Count">
        <property name="minimum">
         <number>3</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="rotZ">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="rotY">
        <property name="text">
         <string>Y</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Radius</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="rotOffset">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="groupingOption">
       <property name="text">
        <string>No grouping</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="acceptButton">
       <property name="text">
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
